backup_and_restore/application_backup_and_restore/troubleshooting.adoc

```diff
@@ -0,0 +1,27 @@
+[id="troubleshooting"]
+= Troubleshooting
+include::modules/common-attributes.adoc[]
+:context: oadp-troubleshooting
+:oadp-troubleshooting:
+:namespace: openshift-adp
+:local-product: OADP
+:must-gather: quay.io/konveyor/oadp-must-gather:latest
```

Comment:
I think we should use downstream oadp-must-gather image

Reply:
I agree. Do you know what it is called?